### PR TITLE
Add confirmations to picture actions

### PR DESCRIPTION
**Pull request by jrmi**

### CODE REVIEW

**Review by vllorca — APPROVED** [review]

**Review by CFarcy — APPROVED** [review]

File: public/locales/fr/translation.json [hunk 1]
```diff
@@ -1008,6 +1018,7 @@
       }
     },
     "usergroup": {
+      "forcedCaseName": "Group",
```
Comment (dtrucs) [inline public/locales/fr/translation.json:1021]: In english?

File: src/modules/RA/Picture/components/CustomToolbar.js [hunk 2]
```diff
@@ -65,6 +65,130 @@ const CancelButton = ({ redirect, className }) => (
...
+        isOpen={open}
+        title={translate('resources.picture.actions.accept_modal_title')}
+        content={translate('resources.picture.actions.accept_modal_content')}
+        onConfirm={handleConfirm}
+        onClose={() => setOpen(false)}
+      />
+    </>
+  );
+};
+
```
Comment (dtrucs) [inline src/modules/RA/Picture/components/CustomToolbar.js:191]: There are a lot of duplication for the same work.
Why don't you pass a boolean prop to display the reason or not?

File: src/modules/RA/Picture/components/PictureFields.js [hunk 3]
```diff
@@ -47,6 +49,16 @@ const useStyles = makeStyles({
       height: 'auto',
       maxHeight: 'inherit',
     },
+    '& .refusal': {
+      position: 'absolute',
+      bottom: '1em',
+      width: '100%',
+      padding: '2em',
```
Comment (dtrucs) [inline src/modules/RA/Picture/components/PictureFields.js:56]: Maybe this element will go outer the width => 100% + 2em + 2em.
The safe workaround in absolute element is to not set a `width: 100%` but `right: 0; left: 0` so you can set the padding you want without side effect

**Review by dtrucs — CHANGES_REQUESTED** [review]

File: public/locales/fr/translation.json (hunk 1 above)
Comment (jrmi) [reply public/locales/fr/translation.json]: Good point :D

File: src/modules/RA/Picture/components/CustomToolbar.js (hunk 2 above)
Comment (jrmi) [reply src/modules/RA/Picture/components/CustomToolbar.js:69]: Refactored

File: src/modules/RA/Picture/components/PictureFields.js (hunk 3 above)
Comment (jrmi) [reply src/modules/RA/Picture/components/PictureFields.js]: Done

Comment (dtrucs) [reply src/modules/RA/Picture/components/PictureFields.js]: Almost, you kept the width :D

**Review by dtrucs — APPROVED** [review]

File: src/modules/RA/Picture/components/PictureFields.js (hunk 3 above)
Comment (jrmi) [reply src/modules/RA/Picture/components/PictureFields.js]: Done

**Pull request merged**